nextflow_schema.json

```diff
@@ -539,6 +551,26 @@
                     "default": "bam_stat,inner_distance,infer_experiment,junction_annotation,junction_saturation,read_distribution,read_duplication",
                     "fa_icon": "fas fa-chart-pie",
                     "description": "Specify the RSeQC modules to run."
+                },
+                "contaminant_screening": {
+                    "type": "string",
+                    "description": "Tool to use for detecting contaminants in unaligned reads - available options are 'kraken2' and 'kraken2_bracken'",
+                    "fa_icon": "fas fa-virus-slash",
+                    "enum": ["kraken2", "kraken2_bracken"]
+                },
+                "kraken_db": {
+                    "type": "string",
+                    "description": "Database when using Kraken2/Bracken for contaminant screening.",
```

Comment:
@egreenberg7 any idea about that?

Reply:
I'll make sure to include the answer to that when we make tests for this new feature in the next release

Reply:
Should be a tar.gz file since the nf-core Kraken2 and Bracken modules use the `--gzip-compressed` option. It probably should be clarified that kraken2 and bracken databases are distinct, but they are often zipped together (as in the pre-built indices). A separate `--bracken_db` option potentially could be added.

While the pre-built files are not gold standard, they provide a quick/convenient option for people who want to run contamination detection without getting into the weeds of the tools. I'll note that I did not offer the ability to make a standard database in an indexing run of the pipeline because it is so much slower than just downloading pre-built indices (and rather computationally expensive). 

In any case, @jfy133 has much more experience than me with metagenomics tools, so I would defer to any changes he recommends to either the Usage tab or the input